ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_utility.py

```diff
@@ -0,0 +1,51 @@
+# local
+import ivy_tests.test_ivy.helpers as helpers
+from ivy_tests.test_ivy.helpers import handle_test
+
+
+@handle_test(
+    fn_tree="functional.ivy.experimental.optional_get_element",
+    dtype_and_x=helpers.dtype_and_values(
+        dtype=[
+            "int8",
+            "int16",
+            "int32",
+            "int64",
+            "complex64",
+            "complex128",
+            "bool",
+            "float16",
+            "float32",
+            "float64",
+            "string",
+            "uint8",
+            "uint16",
+            "uint32",
+        ],
+        min_value=-100,
+        max_value=100,
+        min_num_dims=0,
+        max_num_dims=5,
+        min_dim_size=0,
+        max_dim_size=5,
+        allow_nan=True,
```

Comment:
I would suggest if you removed all of those constraints. As the input can be any tensor type, or list of tensors.